### PR TITLE
Quota view revamp

### PR DESCRIPTION
**Pull request by nobuti**
Working on the acceptance of #11662, I saw the quota view stuck sometimes. After digging a bit, I realized there was a race condition that provoked this. This PR splits the analysis controls view in two separated views that are in charge of their own state.

### CODE REVIEW

**Comment by xavijam** [comment]
_NEWs update not needed_

File: lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/georeference-form-model.js [hunk 1]
```diff
@@ -246,6 +246,7 @@ module.exports = BaseAnalysisFormModel.extend({
     this.clear({silent: true});
     this.set('type', attrs.type, {silent: true}); // re-set type to avoid change:type event to trigger again
     this.set(attrs);
+    this.trigger('change', this);
```
Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/georeference-form-model.js]: Do we need this after doing a `set` (without a silent true)?

Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/georeference-form-model.js]: Fixed

**Comment by xavijam** [comment]
Frontend passed, merging! 💃

**Pull request merged**